CHANGELOG

```diff
@@ -0,0 +1,23 @@
+Changelog
+
+This is the changelog of the metadata manager. All notable changes will 
+be documented in this file. 
+
+[unreleased]
+Added
+- Album cover and icon setter
+- More file name support. Artist can now appear on either side of a -
+- Seperate function to get the song name and artist from
+  various combinations and confirm from spotify database
+- Song without artist name support
+
+[0.1.0] - 2017-03-20
```

Comment:
Date should be 2018, I guess.

Reply:
Oops. Fixed along with cases: https://github.com/fossdelhi/sync-music/commit/39337ba2c2c41faa71d1a46eb08a604a07052650